src/components/Layout/MainLayout.jsx

```
@@ -1934,12 +1938,16 @@ const MainLayout = (props) => {
                                 src={Assets.gameLost}
                                 alt="gameLost"
-                                height={"250px"}
+                                style={{
+                                  height: isMobile ? "220px" : "360px",
+                                  width: "auto",
+                                  objectFit: "contain",
+                                }}
                               />
                               <Typography
                                 sx={{ mb: 1, mt: 1, textAlign: "center" }}
                               >
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)